BuildScripts/mvComplexFunctions.sql

```diff
@@ -1308,28 +1310,29 @@ BEGIN
     aPgMview    		 := mv$getPgMviewTableData( pConst, pOwner, pViewName );
     aPgMviewOjDetails    := mv$getPgMviewOjDetailsTableData( pConst, pOwner, pViewName, pTableAlias);
 	
-	tFromClause		  	 := pConst.FROM_COMMAND  || aPgMview.table_names    || pConst.WHERE_COMMAND;
+	--tFromClause		  	 := pConst.FROM_COMMAND  || aPgMview.table_names    || pConst.WHERE_COMMAND;
 	--tDeleteFromClause := pConst.FROM_COMMAND  || aPgMview.table_names    || pConst.WHERE_COMMAND;
-	--tInsertFromClause := pConst.FROM_COMMAND  || aPgMviewOjDetails.join_replacement_from_sql    || pConst.WHERE_COMMAND;
+	tInsertFromClause := pConst.FROM_COMMAND  || aPgMviewOjDetails.join_replacement_from_sql    || pConst.WHERE_COMMAND;
 
     IF LENGTH( aPgMview.where_clause ) > 0
     THEN
 		tFromClause := tFromClause      || aPgMview.where_clause    || pConst.AND_COMMAND;
```

Comment:
--tFromClause

Reply:
i've commented this line out as i'm not using tFromClause anymore

Reply:
wanted to keep it in just incase i need to revert it back when it gets released into prods - shouldn't need too but you never know